platform/android/export/export_plugin.cpp

```diff
@@ -1670,14 +1670,7 @@ Vector<String> EditorExportPlatformAndroid::get_enabled_abis(const Ref<EditorExp
 }
 
 void EditorExportPlatformAndroid::get_preset_features(const Ref<EditorExportPreset> &p_preset, List<String> *r_features) const {
-	String driver = ProjectSettings::get_singleton()->get("rendering/driver/driver_name");
-	if (driver == "opengl3") {
-		r_features->push_back("etc");
-	}
-	// FIXME: Review what texture formats are used for Vulkan.
-	if (driver == "vulkan") {
-		r_features->push_back("etc2");
-	}
+	r_features->push_back("etc2");
```

Comment:
Shouldn't we have `etc` for the `compatibility/GLES3` driver?

Reply:
OpenGL ES 3.0, OpenGL 3,3, and WebGL 2 all support ETC2 which is a superset of ETC. So essentially we can treat everything as if it is ETC2. Internally we no longer check for ETC support, just ETC2